tee-worker/litentry/core/assertion-build/src/dynamic/contracts/token_holding_amount/BRC20.sol

```diff
@@ -21,62 +21,59 @@ pragma solidity ^0.8.8;
 import "../libraries/Identities.sol";
 import "../libraries/Http.sol";
 import "../libraries/Utils.sol";
-import {TokenHoldingAmount} from "./TokenHoldingAmount.sol";
+import { TokenHoldingAmount } from "./TokenHoldingAmount.sol";
+contract BRC20 is TokenHoldingAmount {
```

Comment:
Can this be abstract ? 

Is this code meant to be uploaded or `BRC20Mapping.sol` ?

Reply:
fixed.